war/src/main/less/pluginSetupWizard.less

```diff
@@ -970,7 +970,7 @@ body .plugin-setup-wizard { // need specificity, revisit the CSS inclusion order
 }
 
 .bootstrap-3 {
-    @import (less) "../../../node_modules/bootstrap/less/bootstrap.less";
+    @import (less) "~bootstrap/less/bootstrap.less";
```

Comment:
this should not have been loading directly from node_modules, at least since webpack was used, this resolves it from the library path.

Reply:
> this should not have been loading directly from node_modules, at least since webpack was used, this resolves it from the library path.

Does it?
![Screenshot 2022-07-24 at 19 22 11](https://user-images.githubusercontent.com/13383509/180658812-5fbb4d02-01a3-4b32-8950-eabb6f43e8bd.png)

Reply:
Yes with webpack you should be using the resolver to load it not hard coding a path

Reply:
bootstrap 4 switched to sass instead of less.

Reply:
> bootstrap 4 switched to sass instead of less, likely related to that.

IJ just acting up again, looks like it refuses to understand the syntax and resolves the path against the directory you are in.
In terms of the bootstrap PR, I adjusted it to `@import (css) "~bootstrap/scss/bootstrap.scss";` locally. Seems to work out well